### ISSUE TITLE
The need for consensus makes closing needlessly slower

### ISSUE THREAD

**Issue by 44castle** [Tiny-Giant/Meta-Drafts#1]
If the goal of closing questions is to give guidance to the asker on how to improve the question, why must there be so many hoops to jump through in order to get that guidance to them? I agree that the user should be able to see all of the reasons why people voted to close a question, and that close voters should be able to select multiple reasons for their close vote, but I think only 1 person choosing a certain close reason should trigger the message to show up (even if that message is very brief). It shouldn't ever take more than 5 people to close a question.

**Comment by Tiny-Giant** [comment]
I'm open to discussion on lowering the proposed number of close votes needed to invoke a closure status.  The reason that I proposed five, is because that's what it is now. Personally I would like to see it at three.e

**Comment by 44castle** [comment]
Perhaps a compromise would be to set a minimum number of people (somewhere from 1-3) to invoke a close status, but still require at least 5 total people to close the question? For example: of the 5 people who vote to close a question, 4 votes went to too-broad, 3 votes went to opinion-based, 1 vote went to no MCVE. Only the too-broad and opinion-based votes will show up, but neither show up until there's 5 people who voted.

**Comment by Tiny-Giant** [comment]
Hmmm... I think that might be plausible. Will sleep on it.

**Comment by makyen** [comment]
I also agree with this. Reading through the proposal and the comments in SOCVR, this was the issue which most stood out to me. Requiring there to be some number of votes on a specific reason encourages people to mark reasons with which they don't, necessarily, agree, just to close the question.

Closure should require at most 5 people, even if they all select different reasons. However, the proposal should not make getting a question closed *harder*, which requiring 5 close votes with the same reason does do.

Perhaps, it can be closed with fewer people if enough (3?,4?) all select the same reason. I'm not sure on that second part. That might be appropriate to put into a "possible further discussion" section. Such a section could list issues that might be revisited in the future when more data is available, either before or after the proposal is implemented.

**Comment by Tiny-Giant** [comment]
I think that the best option is to lower the requirement to three close votes. This removes complexity, and allows questions to be closed and–once the question is edited–reopened more easily, which I believe will promote community involvement in question improvement. 

That said, this proposal is not about me, it is about the community, so I will go with whatever the community decides is best.
